app/models/client_opportunity_match.rb

```diff
@@ -673,6 +676,9 @@ def activate!(touch_referral_event: true, user: nil)
     self.class.transaction do
       update!(active: true)
       client.make_unavailable_in(match_route: match_route, expires_at: nil, user: user, match: self, reason: UnavailableAsCandidateFor::ACTIVE_MATCH_TEXT) if match_route.should_prevent_multiple_matches_per_client
+      match_route.routes_parked_on_active_match.reject(&:empty?).each do |park_route|
```

Comment:
```suggestion
      # Park client on any other routes where this route is set to block matching when the client is successful on this route
      match_route.routes_parked_on_active_match.reject(&:empty?).each do |park_route|
```

Reply:
This has been included in the lates commit